consoleApp/src/main/java/StoreApp.java

```diff
@@ -0,0 +1,12 @@
+import listCategory.Store;
+
+import java.io.IOException;
+
+public class StoreApp {
+
+    public static void main(String[] args) throws IOException {
+
+        Store store = new Store();
+        store.handListCategory();
```

Comment:
handle not a very suitable name for method that should pring store(categories/...)